interfaces/cython/SConscript

```diff
@@ -76,11 +78,226 @@ mod = build(localenv.Command('#build/python/cantera/__init__.py', 'setup.py',
 env['python_module'] = mod
 env['python_extension'] = ext
 
+UNITS = {
```

Comment:
What is added here here is sufficiently complex to warrant an explanation of what is actually being done here. How does this work? If anyone needs to add a new property, how can this be extended (e.g. for `Transport`)